content/docs/deploy/enterprise/upgrading.mdx

```diff
@@ -7,6 +7,34 @@ description: This page contains the list of deprecations and important or breaki
 
 Review the upgrade guide below before upgrading to the latest version of Pomerium Enterprise.
 
+## 0.23.0
+
+### Before you upgrade
+
+#### New
+
+**Set Request Headers**<br/>
+
+[Set Request Headers](/docs/reference/routes/headers#set-request-headers) has a new token substitution value that sends the client certificate fingerprint (downstream mTLS must be enabled) to the upstream application or service.
```

Comment:
Are new features typically mentioned in the upgrading guide? I wonder if it's sufficient just to mention these in the changelog.

Reply:
In the past, they've called out new features and also how to upgrade. I'm fine removing, but would like to see what @desimone thinks before we remove it.

Reply:
@desimone I cleaned this up. Please let me know if the changes are OK!